.github/workflows/reusable-e2e.yml

```
@@ -228,6 +234,22 @@ jobs:
       - name: Pull down Datastore AWS environments
         run: yarn environments datastore pull
 
+      - name: Configure liveness credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-2
```

```suggestion
          aws-region: us-east-1
```

Noticed that this app is in `us-east-1`, not `us-east-2` like other environments. It doesn't have to be this PR since this PR is already large, but we should migrate this environment to `us-east-2` for the sake of consistency.